Source/WebKit/UIProcess/WebAuthentication/fido/CtapHidDriver.cpp

```diff
@@ -194,6 +196,9 @@ void CtapHidDriver::transact(Vector<uint8_t>&& data, ResponseCallback&& callback
 
     auto initCommand = FidoHidMessage::create(m_channelId, FidoHidDeviceCommand::kInit, m_nonce);
     ASSERT(initCommand);
+
```

Comment:
This needs to be moved farther up in the function because data got moved on line 184 (or we could use `m_requestData`.)